README.md

```diff
@@ -28,124 +54,44 @@ from float8_experimental.float8_linear_utils import (
 )
 from float8_experimental.float8_linear import Float8Linear
 
-# create fp32 model
+# create model
 m = Model(...)
 
 # convert all `torch.nn.Linear` modules to `Float8Linear`
 swap_linear_with_float8_linear(m, Float8Linear)
 
+# optional: use FSDP. Note that workarounds are needed for autocast+compile+FSDP+float8 to work
+from float8_experimental import config
+config.enable_amax_init = False
+config.enable_pre_and_post_forward = False
+model = FSDP(model, use_orig_params=True)
+
+# optional: enable torch.compile for improved performance
+m = torch.compile(m)
+
 # toy training loop
 for _ in range(N_ITER):
     optimizer.zero_grad()
     y = m(x)
     y.sum().backward()
 
-    # specific to float8: separate step to sync scales/amaxes
+    # specific to float8 with delayed scaling: separate step to sync scales/amaxes
     # in the future, this may move to a context manager
     sync_float8_amax_and_scale_history(model)
 
     optimizer.step()
 ```
 
-## multi GPU
-
-```python
-from float8_experimental.tp_linear import swap_tp_linear_with_float8_linear
-
-# swaps the fairscale `ColumnParallelLinear` with `Float8ColumnParallelLinear`,
-# and the fairscale `RowParallelLinear` with `Float8RowParallelLinear`
-swap_tp_linear_with_float8_linear(model)
-
-# if applicable, enable sequence parallel on the right modules
-# TODO make the API for this nicer
-model.foo.bar.fc1.sequence_parallel = True
-model.foo.bar.fc2.sequence_parallel = True
-
-# the rest of the flow is the same as the single GPU flow
-```
-
-# high level technical design
```

Comment:
same with this, I don't mind the high level design

Reply:
I think all of the below is slightly outdated and also more of dev docs than user README.  I think we can have an issue about high level design, it would be nice to edit that without OSS->Meta PR sync.  I can copy-paste this there.